### PR TITLE
Create support image  with default and all patterns on sles15sp3

### PR DESCRIPTION
**Pull request by hjluo**
- Description:
   * Create support image autoyast with default modules and all patterns on sles15sp3.
- Related ticket: [**progress_132704**](https://progress.opensuse.org/issues/132704)
- Related MR: [**qam_612**](https://gitlab.suse.de/qa-maintenance/qam-openqa-yml/-/merge_requests/612)
- Needles: N/A
- Verification run: 
  * [**15sp3_all_patterns_VR**](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP3&build=GM&groupid=220)

### CODE REVIEW

File: data/yam/autoyast/support_images/sles15sp3_install_gnome_all_patterns_aarch64.xml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+          <create_subvolumes t="boolean">true</create_subvolumes>
+          <filesystem t="symbol">btrfs</filesystem>
+          <format t="boolean">true</format>
+          <mount>/</mount>
+          <mountby t="symbol">uuid</mountby>
+          <partition_id t="integer">131</partition_id>
+          <partition_nr t="integer">2</partition_nr>
+          <quotas t="boolean">true</quotas>
+          <resize t="boolean">false</resize>
+          <size>83615547392</size>
```
Comment (lemon-suse) [inline data/yam/autoyast/support_images/sles15sp3_install_gnome_all_patterns_aarch64.xml:49]: The size should be 'max' while not a number.

File: data/yam/autoyast/support_images/sles15sp3_install_gnome_all_patterns_aarch64.xml [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+        <min/>
+        <warn/>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>0</uid>
+      <user_password>$6$lhSzyJt4M/UiQ7Q2$bR6EOCkjAi4p8Ve2Vc1dTv8eRoe6HTsvOrAggFlaWakbyd90QhXIxUuN2vrufEGDDjDLvh2ZR8ZhJRQhOP89I0</user_password>
+      <username>root</username>
+    </user>
+  </users>
+</profile>
```
[the remark this answers is not in the document]
Comment (hjluo) [reply data/yam/autoyast/support_images/sles15sp3_install_gnome_all_patterns_aarch64.xml:197]: changed to <200 loc.

File: data/yam/autoyast/support_images/sles15sp3_install_gnome_all_patterns_aarch64.xml (hunk 1 above)
Comment (hjluo) [reply data/yam/autoyast/support_images/sles15sp3_install_gnome_all_patterns_aarch64.xml]: changed.

**Comment by hjluo** [comment]
@lemon-suse  I've added your comments and rerun all the VRs. can you please take a look again? thanks

**Comment by hjluo** [comment]
@lemon-suse  I've addressed all your comments and please review it again. thanks

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**